pkg/config/config.go

```diff
@@ -102,3 +111,60 @@ func (c *Config) getVfInfo(vfPci string) (string, int, error) {
 
 	return pf, vfID, nil
 }
+
+// handleBridgeConfig checks CNI bridge configuration and set ActualBridge options for PluginConfig.
+// If config.Bridge option is empty, config.ActualBridge will be the value of DefaultBridge const.
+// If config.Bridge option contains one bridge name, config.ActualBridge will be that bridge.
+// If config.Bridge option contains a list of bridges, then auto-detect logic will be used,
+// and the function will iterate over configured bridges and check to which bridge uplink is connected.
+// Uplink should be added to one of the bridges when multiple bridges are set in config,
+// this is required for auto-detect logic.
+// When only one bridge is configured there will be no validation that uplink is a part of a bridge,
```

Comment:
`When only one bridge is configured` this may be misunderstood as only one bridge configured in the system.

maybe reword it: `When a single bridge is specified in plugin configuration`

Reply:
done